functional_tests/steps/cms_login.py

```diff
@@ -0,0 +1,43 @@
+from behave import given, then, when  # pylint: disable=E0611
+from behave.runner import Context
+from playwright.sync_api import expect
+
+from functional_tests.step_helpers.users import create_cms_admin_user
+
+
+@given("the user is a CMS admin")  # pylint: disable=E1102
+def user_is_cms_admin(context: Context) -> None:
+    """Create an admin user."""
+    context.username, context.password = create_cms_admin_user()
+
+
+@when("the user navigates to the beta CMS admin page")  # pylint: disable=E1102
+def cms_admin_navigates_to_beta_homepage(context: Context) -> None:
+    """Navigate to admin login page."""
+    context.page.goto(f"{context.base_url}/admin/login/")
+
+
+@when("they enter a their valid username and password and click login")  # pylint: disable=E1102
+def enter_a_valid_username_and_password_and_sign_in(context: Context) -> None:
+    """Enter the username and password and click login."""
```

Comment:
@MebinAbraham @AdamHawtin  An example of the docstrings madness all the step definitions are like this, since the linter (I think ruff) enforces this.

The given step decorator has "they enter a their valid username and password and click login".

The function is named "enter_a_valid_username_and_password_and_sign_in".

Then we have a docstring that says "Enter the username and password and click login".

This is literally weird, and soon this will swirl into madness.

Not sure what everyone else thinks.

Reply:
The requirement for all functions needing docstring has been disabled in the Bundles PR.